src/lib/utils.ts

```diff
@@ -96,3 +97,8 @@ export const getTrimmedTextContent = (
   }
   return node.querySelector(querySelector)?.textContent?.trim()
 }
+
+export const loadJsonData = (filePath: string) => {
+  const rawData = fs.readFileSync(filePath, 'utf-8')
```

Comment:
can we avoid using the file system as that will not work with FE, I know that method will be called just from pressroom but the `utils.ts` will be loaded when we use transform in the FE. you can import `core-sections.json` directly

Reply:
this would require setting `resolveJsonModule` to `true` in `tsconfig.json` 
is that okay?

Reply:
yes that is ok, @mnatsheh @hhusban do you have any objections about `resolveJsonModule`